lib/validation/rules/required.ts

```diff
@@ -1,8 +1,14 @@
-import { required as baseRequired, helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
+import { required as baseRequired } from '../validators'
+
+export const message = {
```

Comment:
this will cause issues, as we are re-exporting everything from /rules/index.ts (every rule will be exporting messages)

Reply:
Ah, you are right. Hmmmm... It might be useful to have messages exported so...

Should we change `index.ts` to selectively export only the function? Then when we want to access the message, we can directly import { message } from `../{rule}` 👀

Reply:
We should actually remove barrel exports. It messes up vite (slow HMR + bad treeshaking).

Reply:
Yeah. I'll do that 👀

Reply:
```ts
export { and, not, or } from '@vuelidate/validators'
export { checked } from './checked'
export { decimal } from './decimal'
export { decimalOrHyphen } from './decimalOrHyphen'
export { email } from './email'
export { fileExtension } from './fileExtension'
export { hms } from './hms'
export { maxFileSize } from './maxFileSize'
export { maxLength } from './maxLength'
export { maxTotalFileSize } from './maxTotalFileSize'
export { maxValue } from './maxValue'
export { minLength } from './minLength'
export { minValue } from './minValue'
export { month } from './month'
export { negativeInteger } from './negativeInteger'
export { positiveInteger } from './positiveInteger'
export { required } from './required'
export { requiredHms } from './requiredHms'
export { requiredIf } from './requiredIf'
export { requiredYmd } from './requiredYmd'
export { rule } from './rule'
export { url } from './url'
export { ymd } from './ymd'
export { zeroOrNegativeInteger } from './zeroOrNegativeInteger'
export { zeroOrPositiveInteger } from './zeroOrPositiveInteger'
```

paste this 😅

Reply:
Done!